proposals/2024-06-14_mixxx_e_v.md

```diff
@@ -0,0 +1,535 @@
+   gewährt. Zu den Kriterien gehören langfristiges Engagement und Beiträge.
+3. Ein Kernmitglied kann in den Status eines außerordentlichen Mitglieds
+   wechseln, indem es einen Antrag an den Vorstand stellt. Bei Nichterfüllung
+   der oben angegebenen Pflichten eines aktiven Mitglieds über zwei
+   aufeinanderfolgende ordentliche Mitgliederversammlungen ändert sich die
+   Mitgliedschaft automatisch in eine außerordentliche.
+4. Die Mitgliedschaft endet durch Austritt, Ausschluss oder Tod, bei Verlust
+   der Rechtspersönlichkeit. Der Austritt aus dem Verein wird zum Ende des
+   folgenden Kalendermonats wirksam, in dem die Austrittserklärung
+   eingereicht wurde.
+5. Der Ausschluss eines Mitglieds kann von der Mitgliederversammlung oder durch
+   eine Internetabstimmung beschlossen werden, wenn eine kritische Situation
+   vorliegt und eine gütliche Lösung nicht erreicht werden kann. Die folgenden
+   Gründe können einen Ausschluss bewirken:
+   * a) Schwerwiegende Verstöße gegen die Satzungsbestimmungen,
+     den Zweck und die Ziele des Vereins oder dessen Verhaltenskodex.
+   * b) Nichtbezahlen von Mitgliedsbeiträgen über einen Zeitraum von 12 Monaten.
+   Dem Mitglied muss vor dem Ausschluss die Möglichkeit gegeben werden, sich zu
+   rechtfertigen
+6. Bei Ausscheiden eines Mitglieds aus dem Verein oder bei Vereinsauflösung
+   besteht kein Anspruch auf Rückerstattung etwa eingebrachter Vermögenswerte.
+
+## 5. Organe des Vereins
+
+* die Mitgliederversammlung
+* der Vorstand
+
+## 6. Mitgliederversammlung
+
+1. Der Mitgliederversammlung gehören alle Kernmitglieder, mit je einer Stimme
+   an.
+2. Sie findet mindestens einmal im Jahr statt, wobei die Einladungen mindestens
+   sechs Wochen vorher mit einer vorläufigen Tagesordnung an alle
+   Kernmitglieder und außerordentlichen Mitglieder versendet werden muss. Die
+   Mitglieder können dem Vorstand bis zwei Wochen vor der Versammlung
+   zusätzliche Tagesordnungspunkte einreichen. Der Vorstand veröffentlicht die
+   endgültige Tagesordnung online.
+3. Eine außerordentliche Mitgliederversammlung kann einberufen werden, wenn dies
+   von mindestens 20 % der Kernmitglieder beantragt wird.
+4. Die Mitgliederversammlung ist beschlussfähig, wenn die Einladung in
+   ordnungsgemäßer Form erfolgt ist. Bei Stimmengleichheit gilt ein Antrag als
+   abgelehnt.
+5. Zur Änderung der Satzung oder zur Auflösung des Vereins ist eine 3/4-Mehrheit
+   der Stimmen erforderlich.
+6. Kernmitglieder, die nicht teilnehmen können, können sich durch ein anderes
+   Mitglied mit schriftlicher Vollmacht vertreten lassen.
+7. Die Teilnahme an der Mitgliederversammlung und die Stimmabgabe müssen auch auf
+   elektronischem Wege möglich sein. Durch Beschluss des Vorstandes kann eine
+   Mitgliederversammlung ausschließlich auf diesem Wege abgehalten werden.
+
+## 7. Aufgaben der Mitgliederversammlung
+
+1. Die Generalversammlung ist das höchste beschlussfassende Organ und ist
```

Comment:
```suggestion
1. Die Mitgliederversammlung ist das höchste beschlussfassende Organ und ist
```
We should use the same term accross the document. Generalversammlung or Jahreshauptversammlung is also fine. Generalversammlung / "General assembly"  might be better understandable in the English translation.

Reply:
This is the term KDE uses. But I don't mind to change it.